site/docs/reference/connectors-storagemanager/README.md

```diff
@@ -0,0 +1,41 @@
+# Storage Manager Service
+
+<a name="documentation-for-api-endpoints"></a>
+## Documentation for API Endpoints
+
+All URIs are relative to *https://localhost:8082*
+
+Class | Method | HTTP request | Description
+------------ | ------------- | ------------- | -------------
+*DefaultApi* | [**allocateStorage**](Apis/DefaultApi.md#allocatestorage) | **POST** /allocateStorage | This REST API allocates storage based on the storage account selected by Fybrik
+*DefaultApi* | [**deleteStorage**](Apis/DefaultApi.md#deletestorage) | **DELETE** /deleteStorage | This REST API deletes allocated storage
+*DefaultApi* | [**getSupportedConnections**](Apis/DefaultApi.md#getsupportedconnections) | **POST** /getSupportedConnections | This REST API returns a list of supported storage types
```

Comment:
maybe it is late now, but can the method name be misleading?
What about `getSupportedStorageTypes`

Reply:
changed